operations/firmware/FAS_CLI.md

```diff
@@ -58,7 +58,7 @@ This covers the generic process for executing an action. For more specific examp
 
    Filters narrow the scope of FAS to target specific component names (xnames), manufacturers, targets, and so on. For this example, FAS will run with no selection filters applied.
 
-1. Create a JSON file.
+1. (`ncn-mw#`) Create a JSON file.
```

Comment:
This is the CSM 1.3 prompt format. 

@mharding-hpe @nrockershousen  I assume that the CSM 1.2 docs should follow this convention https://github.com/Cray-HPE/docs-csm/blob/release/1.2/introduction/documentation_conventions.md#command-prompt-conventions instead of the CSM 1.3 version?

Reply:
This comment applies to all pages change in this PR, as they all seem to have the CSM 1.3 prompt conventions.

Reply:
That is correct. The prompts can stay inside the code blocks for 1.2.